src/main/java/org/jenkinsci/plugins/ewm/definitions/Disk.java

```diff
@@ -32,7 +31,7 @@ public Disk(String diskId, String name, String masterMountPoint, String physical
         this.diskId = fixEmptyAndTrim(diskId);
         this.name = fixEmptyAndTrim(name);
         this.masterMountPoint = fixEmptyAndTrim(masterMountPoint);
-        this.physicalPathOnDisk = fixEmptyAndTrim(physicalPathOnDisk);
+        this.physicalPathOnDisk = fixNull(physicalPathOnDisk).trim();
```

Comment:
It's unrelated to help files, I'd guess. Maybe was better to keep null in fields and process it in the logic